ajax/libs/jQuery.BgSwitcher/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jQuery.BgSwitcher",
+  "filename": "jquery.bgswitcher.min.js",
+  "version": "0.4.3",
+  "description": "Switch the background image with using effect.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rewish/jquery-bgswitcher.git"
+  },
+  "author": "rewish",
```

Comment:
@cdnjs/intern3 a single string sometimes is meaningless and I don't want to you add the single string by yourselves.